src/components/TabPanel.js

```diff
@@ -24,28 +25,37 @@ module.exports = React.createClass({
   getDefaultProps() {
     return {
       selected: false,
+      selectedClassName: null,
+      style: {},
       id: null,
       tabId: null,
     };
   },
 
   render() {
-    const { className, children, selected, id, tabId, style, ...attributes } = this.props;
+    const {
+      id, tabId, selected,
+      className, selectedClassName, style, children,
+      ...attrs,
+    } = this.props;
+
+    // Only hide using inline styles if `selectedClassName` isn't provided
```

Comment:
We could also just follow the design of the `Tab` component which only uses overridable classes.

The case I ran into was that I needed hidden `TabPanel`s to still take up their natural height, i.e. setting `visibility: hidden;` instead on the inline `display: none;`.